src/Layers/xrRenderPC_GL/r3_R_rain.cpp

```diff
@@ -175,16 +173,15 @@ void CRender::render_rain()
         //bb.max.y = 50;
 
         //	Offset RainLight position to center rain shadowmap
-        Fvector3 vRectOffset;
-        vRectOffset.set(fBoundingSphereRadius * Device.vCameraDirection.x, 0,
-                        fBoundingSphereRadius * Device.vCameraDirection.z);
+        Fvector3 vRectOffset = {fBoundingSphereRadius * Device.vCameraDirection.x, 0,
+                                fBoundingSphereRadius * Device.vCameraDirection.z};
         bb.vMin.x = -fBoundingSphereRadius + vRectOffset.x;
         bb.vMax.x = fBoundingSphereRadius + vRectOffset.x;
         bb.vMin.y = -fBoundingSphereRadius + vRectOffset.z;
         bb.vMax.y = fBoundingSphereRadius + vRectOffset.z;
 
         //D3DXMatrixOrthoOffCenterLH	((D3DXMATRIX*)&mdir_Project,bb.min.x,bb.max.x,  bb.min.y,bb.max.y,  bb.min.z-tweak_rain_ortho_xform_initial_offs,bb.max.z);
-        D3DXMatrixOrthoOffCenterLH((D3DXMATRIX*)&mdir_Project, bb.vMin.x, bb.vMax.x, bb.vMin.y, bb.vMax.y,
+        XRMatrixOrthoOffCenterLH(&mdir_Project, bb.vMin.x, bb.vMax.x, bb.vMin.y, bb.vMax.y,
```

Comment:
If this is LH, shouldn't other call to glm::ortho also be LH?

Reply:
No, in this place need glm::orthoLH, but this func available in newer GLM versions